### DISCUSSION TITLE
Will not load

### DISCUSSION THREAD

**Discussion by Turox** [mrlt8/docker-wyze-bridge#794] [Q&A]
when I start wyze bridge docker it runs but when I open webui it's blocked. here is log
[WyzeBridge] ♻️ Clearing local cache...
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
 * Serving Flask app 'frontend'
 * Debug mode: off
[WyzeBridge] WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.17.0.4:5000
[WyzeBridge] Press CTRL+C to quit
[WyzeBridge] ⚠️ 400 Client Error: Bad Request for url: https://auth-prod.api.wyze.com/user/login
[WyzeBridge] {"description":"Invalid User Name or Password.","requestId":"bb516abd-5f24-4bee-882c-6b77cd0fd672","errorCode":1000}
[WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
[WyzeBridge] ⚠️ 400 Client Error: Bad Request for url: https://auth-prod.api.wyze.com/user/login
[WyzeBridge] {"description":"Too many failed attempts. Please try again later.","requestId":"20e90d67-04f1-4c1f-b394-90e41cac1627","errorCode":1000}
[WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
[WyzeBridge] ⚠️ 400 Client Error: Bad Request for url: https://auth-prod.api.wyze.com/user/login
[WyzeBridge] {"description":"Too many failed attempts. Please try again later.","requestId":"a7915404-4de4-4a6f-b503-c661cd73b970","errorCode":1000}
[WyzeBridge] starting MediaMTX v0.22.2
[WyzeBridge] Stopping 0 streams
[WyzeBridge] Stopping MediaMTX...
[WyzeBridge] 👋 goodbye!
 What am I doing wrong? I checked password  and login both are correct. when you go to  
 https://auth-prod.api.wyze.com/user/login it says it is not working. Any ideas ?

**Comment by mrlt8** [comment]
You may need to escape any special characters in your credentials.

**Reply by n2jwc** [reply]
just to be clear, you expect \ as the escape character, correct?

**Reply by mrlt8** [reply]
It depends.. I believe some characters like `$` need to be escaped with an extra character like this `$$`.
